lib/spack/spack/binary_distribution.py

```diff
@@ -2023,7 +2023,7 @@ def install_root_node(spec, allow_root, unsigned=False, force=False, sha256=None
     with spack.util.path.filter_padding():
         tty.msg('Installing "{0}" from a buildcache'.format(spec.format()))
         extract_tarball(spec, download_result, allow_root, unsigned, force)
-        spack.hooks.post_install(spec)
+        spack.hooks.post_install(spec, False)
```

Comment:
This is probably OK. It's also called through `spack buildcache install`, but I think that's more of a legacy command, so let's ignore it.